libs/core/src/components/sage-avatar/stories/sage-avatar.stories.js

```diff
@@ -0,0 +1,46 @@
+import { html } from 'lit-html';
+import imgFile from '../assets/demi_wilkinson.png';
```

Comment:
Nitpick, but could this be converted to a `.jpg` to discourage developers from possibly using a `.png` for non-transparent images/photos?

Reply:
This is good idea. I'm updating this.